src/amo/components/AddonMoreInfo/index.js

```diff
@@ -196,14 +256,16 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
     privacyPolicyLink = null,
     eulaLink = null,
     filesize = null,
+    relatedCategories = null,
     version = null,
     versionLastUpdated,
     versionLicenseLink = null,
     versionHistoryLink = null,
   }: Object) {
-    const { addon, i18n } = this.props;
+    const { addon, i18n, errorHandler } = this.props;
     return (
       <>
+        {errorHandler.renderErrorIfPresent()}
```

Comment:
I'm not sure if this is the best place to render this error or not. I suggest you take a screenshot of how this looks when the error is rendered and attach it to the PR.

Reply:
Ohk, I'll do that!